config/locales/en.yml

```diff
@@ -2927,6 +2927,7 @@ en:
         edit:
           help_text: We will collect any missing information.
           interest_income_body: No additional state info needed
+          interest_income_body_nj: No additional state information needed.
```

Comment:
[dust] I'm surprised this is an NJ-specific change? Did we see if CFA wants to change this across all pages?

Reply:
hm yeah if it's just "info" -> "information" maybe we could change it everywhere...will ask

Reply:
i went ahead and asked in slack, hope that's okay!

Reply:
confirmed with design, it's ok to change everywhere! https://cfastaff.slack.com/archives/C07GYAQ4E1E/p1738699785433089